hasura/hasura-migrations/metadata/tables.yaml

```diff
@@ -199,7 +205,7 @@
         tips:
           arguments:
             where:
-              hash: $onchain_tip_id
+              id: $onchain_tip_id
```

Comment:
why this? I changed it iirc for the tip identifcation by hash to work. Primsa uses numbers as id, we should use the hash because there's no other identifier on chain

Reply:
fixed